content/en/serverless/_index.md

```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
 
 ### 1. Install the AWS Integration
 
-Start by installing the [AWS Integration][2]. This allows Datadog to ingest Amazon CloudWatch metrics from AWS Lambda. The AWS integration installation also configures the Datadog Forwarder, which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+Install the [AWS integration][2]. This allows Datadog to ingest Lambda metrics from AWS CloudWatch. 
 
-### 2. Instrument your application
+### 2. Install the Datadog Forwarder
```

Comment:
```suggestion
### 2. Install the Datadog forwarder
```

Reply:
We do use `Forwarder` in all places wherever it's mentioned in the doc.